compiler/rustc_typeck/src/collect.rs

```diff
@@ -2224,6 +2226,9 @@ fn gather_explicit_predicates_of(tcx: TyCtxt<'_>, def_id: DefId) -> ty::GenericP
         + has_own_self as u32
         + early_bound_lifetimes_from_generics(tcx, ast_generics).count() as u32;
 
+    debug!(?predicates);
```

Comment:
should these be trace too? i think you can also combine them

Reply:
Fixed